javascripts/enemy.js

```diff
@@ -0,0 +1,13 @@
+var FoodFight = (function(enemy) {
+
+  enemy.Player2 = function () {
+    this.health = 50
+    this.attack = 10
+  }
+
+  enemy.player2 = new enemy.Player2();
+  return enemy
+
+})(FoodFight ||{})
+
+console.log(FoodFight.player2);
```

Comment:
if you wouldn't mind sticking a new empty line at the end so github doesn't tell us it's wrong :)